inst/shiny/plotBoxplots.R

```diff
@@ -0,0 +1,84 @@
+plotBoxplots <- reactive({
+  
+  w.use <- w.use()
+  
+  data.elements <- c(df[["data.element"]],df[["data.element.y"]])
+  
+  areas.p2e <- df[["area"]]
+  
+  if(all(df[["areas"]] %in% areas.p2e)){
+    areas.p2e <- NA
+  }
+  notch <- input$notchOn
+  yrs <- NA # may want to code ability to select certain years
```

Comment:
Good. I initially tried including it, but quickly opted to go with NA to just get the new tab to work first! It was an interesting reverse-engineering process mimicking the stuff you'd already put together. Much of it I can follow, some of it I have to admit is still a mystery.